### ISSUE TITLE
Select: Single select control to access modal can't be opened with keyboard (2.1.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#865]
### Expected behavior

Using return key opens the single select modal. 
### Actual behavior

No way to open the single select modal with a keyboard to be able to use component with keyboard.

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.10.2](https://github.com/blackbaud/skyux/releases/tag/v1.10.2)
